docs/README.md

```diff
@@ -95,10 +95,10 @@ kubectl apply -f deploy/kubernetes
 
 Now any user can start creating and using EBS volumes with the CSI driver. 
 
-6. Apply `examples/kubernetes/volume_scheduling` that uses the recently deployed driver:
+6. Apply `examples/kubernetes/dynamic-scheduling` that uses the recently deployed driver:
```

Comment:
Feels to me `volume-scheduling` might be less confusion since `dynamic provisioning` might not have `volume scheduling` enabled.